build.sbt

```diff
@@ -172,30 +189,40 @@ lazy val example = (project in file("example"))
         "com.typesafe.akka"             %% "akka-discovery"                    % AkkaVersion,
         "com.lightbend.akka.management" %% "akka-management"                   % AkkaManagementVersion,
         "com.lightbend.akka.management" %% "akka-management-cluster-http"      % AkkaManagementVersion,
-        "com.lightbend.akka.management" %% "akka-management-cluster-bootstrap" % AkkaManagementVersion,
-        "io.opentelemetry"    % "opentelemetry-sdk-extension-autoconfigure" % OpentelemetryAlphaMinor0Version,
-        "io.grpc"             % "grpc-netty-shaded"                         % "1.53.0",
-        "org.wvlet.airframe" %% "airframe-log"                              % AirframeVersion
+        "com.lightbend.akka.management" %% "akka-management-cluster-bootstrap" % AkkaManagementVersion
       )
     },
-    assemblyMergeStrategySettings,
-    mainClass                  := Some("example.Boot"),
-    assembly / assemblyJarName := "mesmer-akka-example.jar",
-    resolvers += "Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots",
-    run / fork := true,
-    run / javaOptions ++= {
-      val properties = System.getProperties
+    mainClass := Some("example.Boot"),
+    run / javaOptions ++= Seq(
+      s"-Dotel.service.name=mesmer-example",
+      s"-Dotel.metric.export.interval=5000"
+    )
+  )
+  .dependsOn(core)
 
-      import scala.collection.JavaConverters._
-      val keys = for {
-        (key, value) <- properties.asScala.toList if value.nonEmpty
-      } yield s"-D$key=$value"
+lazy val exampleAkkaStream = (project in file("examples/akka-stream"))
+  .settings(exampleCommonSettings)
+  .settings(
+    name := "mesmer-akka-stream-example",
+    libraryDependencies ++= akka,
+    mainClass := Some("example.SimpleStreamExample"),
+    run / javaOptions ++= Seq(
+      s"-Dotel.service.name=mesmer-stream-example",
+      s"-Dotel.metric.export.interval=5000"
+    )
+  )
+  .dependsOn(core)
 
-      keys
-    },
```

Comment:
This feeding of `System.getProperties` to `javaOptions` was breaking the Windows classpath. It does not seem necessary for examples to work.